twisted/test/test_defer.py

```diff
@@ -2357,7 +2357,7 @@ def test_waitUntilLockedWithTimeoutUnlocked(self):
         but the lock is unlocked before our timeout.
         """
         def onTimeout(f):
-            f.trap(defer.TimeoutError)
+            f.trap(error.TimeoutError)
```

Comment:
Good question :)

I would say that we should have a `# pragma: no-cover` comment to let coverage know that this line should be ignored

... but first, let's talk about it over the mailing list. I will start the conversation :)

Thanks!